### DISCUSSION TITLE
Idea: Collection Image Source Image URL

### DISCUSSION THREAD

**Discussion by pateljoel** [directus/directus#11120] [Feature Requests]
This may already exist in directus but not sure where to look, but is it possible to have the 'Image Source' in the layout options to have the ability to use a HTTP/S Image source rather than a manually uploaded image into directus?

**Comment by joselcvarela** [comment]
Right now, all the images should come from Directus.
You can import files via Upload interface:

https://user-images.githubusercontent.com/14039341/149883961-251bcd48-f4ee-4dd4-977c-64740aa6a464.mov

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
